gnd/src/main/java/com/google/android/gnd/repository/DataRepository.java

```diff
@@ -121,7 +122,17 @@ private Completable updateLocalFeature(RemoteDataEvent<Feature> event) {
     Log.d(TAG, " Activating project " + projectId);
     return remoteDataStore
         .loadProject(projectId)
-        .doOnError(e -> Log.e(TAG, "Project not found", e))
+        .flatMap(project -> localDataStore.insertOrUpdateProject(project).toSingleDefault(project))
+        .onErrorResumeNext(
+            throwable -> {
```

Comment:
Pretty sure this can be simplified, let's revisit in a follow-up.

Reply:
Yes. Let's do a bit cleanup sometime soon.